src/app/core/components/language-mismatch/language-mismatch.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { mapPending } from 'src/app/shared/operators/map-pending';
+import { UserSessionService } from 'src/app/shared/services/user-session.service';
+import { LocaleService } from '../../services/localeService';
+
+@Component({
+  selector: 'alg-language-mismatch',
+  templateUrl: './language-mismatch.component.html',
+  styleUrls: [ './language-mismatch.component.scss' ],
+})
+export class LanguageMismatchComponent {
+
+  readonly currentLanguage = this.localeService.currentLang?.tag;
+  readonly languages = this.localeService.languages;
```

Comment:
This seems to be used at only one place, just use `this.localeService.languages` directly on line 23

Reply:
Fixed